pkg/command/context_test.go

```diff
@@ -1877,6 +1878,7 @@ func TestContextCurrentCmd(t *testing.T) {
   Type:            tanzu
   Organization:    org-name (org-id)
   Project:         project-name (project-id)
+  Scope Path:      /project-name
```

Comment:
Should we not use the full format? `/projects/project-name` and same for the other entities?

Reply:
Oops. missed it. Thanks for catching it. Updated